Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla43161.cs

```diff
@@ -56,10 +56,14 @@ protected override void Init()
 			Content = new StackLayout { Children = { label, listView, listView2, listView3 } };
 		}
 
-#if (UITEST && __IOS__)
+#if UITEST && __IOS__
 		[Test]
 		public void Bugzilla43161Test()
 		{
+			// Avoid launch the test (temporarily) in iOS 14 due to a bug sometimes not rendering some cell. 
+			if (RunningApp.IsVersionOrNewer(14))
```

Comment:
well not sure ignoring the test is a good call since  it's actually catching the failure.

Reply:
Updated to avoid ignore this tests in iOS 14.